src/train.py

```diff
@@ -101,14 +123,28 @@ def main(argv):
             labels = [CLASS_INDEX[label] for label in labels]
             labels = torch.tensor(labels).to(DEVICE)
 
-            # Zero the parameter gradients
-            optimizer.zero_grad()
+            style_vector = torch.rand(500, device=DEVICE)
+            char_vector = charVector()
+
+            #the generator is provided by the pantry.py
+            charGenerated = generator.generate(char_vector, style_vector)
 
+            # Zero the parameter gradients #weird investigate further
+            optimizerDisc.zero_grad()
+
+            # Train the optimizer
```

Comment:
We don't train the optimizer! The optimizer is needed to train the network :innocent:

Reply:
i think typo hahaha